ghc-lib/new-working-on-ghc-lib.md

```diff
@@ -65,3 +65,91 @@ Once you have the GHC patch you want to incorporate into the DAML repo, here's t
 3. Run the pin command on linux or mac `bazel run @stackage-unpinned//:pin` and commit those changes as well
 
 4. Before merging the PR, the pin command will also have to be run on windows, and those changes committed as well. You will need access to a windows machine for that: `ad-hoc.sh windows create`
+
+
+### Working on an `add-hoc` windows machine
+
+1. First time, clone the `daml-language-ad-hoc` repo: (On following times, just pull for any updates to the scripts)
+```
+git clone git@github.com:DACH-NY/daml-language-ad-hoc.git
+cd daml-language-ad-hoc
+direnv allow
+```
+
+2. Create a new windows machine:
+```
+./ad-hoc.sh create windows
+```
+
+3. When the script complete, note the IP and Password.
+
+4. Wait at least 10 minutes before trying to remotely connect to the windows machine, or else it may not inialize correctly. Really, I am not kidding. This advice is from Gary!
```

Comment:
Unfortunately true. I'll try to find some time to investigate further over the next few days, but it seems really bizarre so far.